mail-mta/sendmail/sendmail-8.17.1.9.ebuild

```diff
@@ -1,4 +1,4 @@
-# Copyright 1999-2023 Gentoo Authors
+# Copyright 1999-2024 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=8
```

Comment:
See comment below. Why didn't you bump libmilter?

Reply:
I was bumping sendmail because of a security bug but I can bump libmilter as well.

Reply:
Yes, you must. They must be bumped together, or not at all. That's what the comment says, right?

Reply:
Right, I checked and the version in 8.18.0.2 didn't seem to have changed `#  define SMFI_VERSION	0x01000002`, should I revbump it and use the new version to be safe?

Reply:
Diff the tarballs and see if any of the libmilter stuff changed?

Reply:
ping

Reply:
Apologies! I think this got lost in the shuffle of me moving back to college. I'll try to finish it soon.

Reply:
I diff'd the libmilter directories between the two sendmail versions and it doesn't come back with any changes, do I do a revbump for libmilter and increase `SENDMAIL_VER`?